catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/impl/FileSystemPersistenceProvider.java

```diff
@@ -119,6 +119,7 @@ public void store(String key, Object value) {
                 // Intentionally ignored
             }
             IOUtils.closeQuietly(file);
+            IOUtils.closeQuietly(buffer);
```

Comment:
Same comment about finally

Reply:
Updated to use try with resource in each of these files.